code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -328,16 +328,15 @@ All foods are distributed among various categories. Use common sense.
 		if(isdog(M))
 			var/mob/living/L = M
 			if(bitecount == 0 || prob(50))
-				M.manual_emote("nibbles away at \the [src]")
+				M.manual_emote("nibbles away at \the [src].")
 			bitecount++
 			L.taste(reagents) // why should carbons get all the fun?
 			if(bitecount >= 5)
-				var/sattisfaction_text = pick("burps from enjoyment", "yaps for more", "woofs twice", "looks at the area where \the [src] was")
-				if(sattisfaction_text)
-					M.manual_emote(sattisfaction_text)
+				var/satisfaction_text = pick("burps from enjoyment.", "yaps for more!", "woofs twice.", "looks at the area where \the [src] was.")
+				if(satisfaction_text)
+					M.manual_emote(satisfaction_text)
```

Comment:
```suggestion
				M.manual_emote(pick("burps from enjoyment.", "yaps for more!", "woofs twice.", "looks at the area where \the [src] was."))
```

Reply:
the if is unnecessary and adds unnecessary indentation, which we try our best to avoid. You could alternatively still assign ``satisfaction_text`` to a var prior to emoting if you really wanted to, but I don't really see any other reason to do this.